DOCS-8441 LLM Observability Terms and Concepts Update (#24339)
DataDog/documentation: content/en/llm_observability/terms/_index.md

## Evaluations

> LLM Observability offers quality checks and out-of-the-box metrics to evaluate the quality and effectiveness of your LLM conversations, including assessments of sentiment, topic relevancy, and user satisfaction. With evaluations, you can understand the performance of conversations and enhance your LLM application's responses which improves the user experience and ensures valuable, accurate outputs.

Reviewer: ```suggestion
LLM Observability offers quality checks and out-of-the-box metrics to evaluate the quality and effectiveness of your LLM conversations, including assessments of sentiment, topic relevancy, and user satisfaction. With evaluations, you can understand the performance of conversations and enhance your LLM application's responses. This improves the user experience and ensures valuable, accurate outputs.
```

{{< img src="llm_observability/evaluations/evaluations.png" alt="A quality evaluation in LLM Observability" style="width:100%;" >}}

In addition to evaluating conversations, LLM Observability integrates with [Sensitive Data Scanner][5], which helps prevent data leakage by identifying and flagging any sensitive information (such as personal data, financial details, or proprietary information) that may be present in conversations. 

By proactively scanning for sensitive data, LLM Observability ensures that conversations remain secure and compliant with data protection regulations. This additional layer of security reinforces Datadog's commitment to maintaining the confidentiality and integration of user interactions with LLMs.

LLM Observability associates evaluations with individual spans so you can view the inputs and outputs that led to a specific evaluation. While Datadog provides out-of-the-box evaluations for your traces, you can also [submit your own evaluations][6] to LLM Observability.

### Quality evaluations

#### Topic Relevancy

> This check identifies and flags user inputs that deviate from the configured acceptable input topics, ensuring that interactions stay pertinent to the LLM's designated purpose and scope.

Reviewer: ```suggestion
This check identifies and flags user inputs that deviate from the configured acceptable input topics. This ensures that interactions stay pertinent to the LLM's designated purpose and scope.
```

{{< img src="llm_observability/evaluations/sentiment.png" alt="A Sentiment evaluation detected by an LLM in LLM Observability" style="width:100%;" >}}

| Evaluation Stage | Evaluation Method | Evaluation Definition | 
|---|---|---|
| Evaluated on Input and Output | Evaluated using LLM | Sentiment flags the emotional tone or attitude expressed in the text, categorizing it as positive, negative, or neutral.   |

### Security and Safety evaluations

#### Toxicity

This check evaluates each input prompt from the user and the response from the LLM application for toxic content. This check identifies and flags toxic content to ensure that interactions remain respectful and safe.

{{< img src="llm_observability/evaluations/toxicity.png" alt="A Toxicity evaluation detected by an LLM in LLM Observability" style="width:100%;" >}}
  
| Evaluation Stage | Evaluation Method | Evaluation Definition | 
|---|---|---|
| Evaluated on Input and Output | Evaluated using LLM | Toxicity flags any language or behavior that is harmful, offensive, or inappropriate, including but not limited to hate speech, harassment, threats, and other forms of harmful communication. |

#### Prompt Injection

> This check identifies attempts to manipulate the LLM's responses or redirect the conversation in ways not intended by the original users. This check maintains the integrity and authenticity of interactions between users and the LLM.

Reviewer: Is this about a third party or is this about the original intent by the designer? I'm a bit confused.

> This check identifies attempts by unauthorized or malicious users to manipulate the LLM's responses or redirect the conversation in ways not intended by the original user. This check maintains the integrity and authenticity of interactions between users and the LLM.

Author: ```suggestion
This check identifies attempts by unauthorized or malicious authors to manipulate the LLM's responses or redirect the conversation in ways not intended by the original author. This check maintains the integrity and authenticity of interactions between users and the LLM.
```

{{< img src="llm_observability/evaluations/prompt_injection.png" alt="A Prompt Injection evaluation detected by an LLM in LLM Observability" style="width:100%;" >}}

| Evaluation Stage | Evaluation Method | Evaluation Definition | 
|---|---|---|

> | Evaluated on Input | Evaluated using LLM | Prompt Injection Flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or user. |

Author: ```suggestion
| Evaluated on Input | Evaluated using LLM | Prompt Injection flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or user. |
```

> | Evaluated on Input | Evaluated using LLM | Prompt Injection flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or user. |

Author: ```suggestion
| Evaluated on Input | Evaluated using LLM | Prompt Injection flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or author. |
```

> | Evaluated on Input | Evaluated using LLM | Prompt Injection flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or author. |

Author: ```suggestion
| Evaluated on Input | Evaluated using LLM | Prompt Injection flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or user. |
```

#### Sensitive Data Scanning

This check ensures that sensitive information is handled appropriately and securely, reducing the risk of data breaches or unauthorized access.

> {{< img src="llm_observability/evaluations/sensitive_data_scanning.png" alt="A Security & Safety evaluation detected by the Sensitive Data Scanner in LLM Observability" style="width:100%;" >}}

Reviewer: ```suggestion
{{< img src="llm_observability/evaluations/sensitive_data_scanning.png" alt="A Security and Safety evaluation detected by the Sensitive Data Scanner in LLM Observability" style="width:100%;" >}}
```

Reviewer: (unless this is a specific term we always use with an ampersand)
